tuftool/src/error.rs

```diff
@@ -260,6 +270,10 @@ pub(crate) enum Error {
         backtrace: Backtrace,
     },
 
+    /// Root creates an unloadable repo
+    #[snafu(display("Unstable root, less keys then threshold for: {}", role))]
```

Comment:
Suggestion:

`"Unstable root: '{}' role contains {} keys, threshold is {}", role, actual, threshold`